azure-pipelines.yml

```diff
@@ -35,10 +35,14 @@ stages:
   pool:
     vmImage: 'windows-latest'
   jobs:
+  - job: SynapseSearchValidate
```

Comment:
Maybe `E2eTests`?

Reply:
Updated to SynapseE2ETest, the yml script been renamed to **Synapse-E2E-Test.yml**